src/screens/Messages/List/ChatListItem.tsx

```diff
@@ -208,7 +208,7 @@ function ChatListItemReady({
           !isDeletedAccount
             ? _(msg`Go to conversation with ${profile.handle}`)
             : _(
-                msg`The recipient of this chat has deleted their account. Click for options.`,
+                msg`The recipient of this chat has deleted their account. Press for options.`,
```

Comment:
Apologies for being pedantic, but aren't there also cases where the account may have been taken down by moderators or deactivated (which I'm assuming has the same result in terms of the chat being inaccessible)?

What about something like:

"**This chat is no longer available as the recipient's account has been deleted or deactivated. Press for options.**"

Reply:
Good call!